saleor/dashboard/collection/views.py

```diff
@@ -0,0 +1,67 @@
+from django.template.response import TemplateResponse
+from django.contrib import messages
+from django.contrib.auth.decorators import permission_required
+from django.core.urlresolvers import reverse
+from django.http import JsonResponse
+from django.shortcuts import redirect, get_object_or_404
+from django.utils.translation import pgettext_lazy
+
+from ...product.models import Collection
+from ...core.utils import get_paginator_items
+from ..views import staff_member_required
+from .forms import CollectionForm
+
+
+@staff_member_required
+@permission_required('product.view_product')
+def collection_list(request):
+    collections = Collection.objects.prefetch_related('products').all()
+    collections = get_paginator_items(collections, 30, request.GET.get('page'))
+    ctx = {'collections': collections}
+    return TemplateResponse(request,
+                            'dashboard/collection/list.html', ctx)
+
+
+@staff_member_required
+@permission_required('product.edit_product')
+def collection_create(request):
+    collection = Collection()
+    form = CollectionForm(request.POST or None)
```

Comment:
Missing `instance=collection` in the form initialization. I'm not sure now if this is required for a ModelForm to create an object properly, but we follow this rule in other views as well.